examples/drivers/windows/otLwf/filter.h

```diff
@@ -111,6 +111,29 @@ typedef enum OTLWF_DEVICE_STATUS
 #define OT_EVENT_TIMER_RUNNING      1
 #define OT_EVENT_TIMER_FIRED        2
 
+#if OPENTHREAD_CONFIG_PLATFORM_MESSAGE_MANAGEMENT
+
+typedef struct BufferPool
+{
+    struct BufferPool* Next;
+    uint8_t Buffers[0];
+
+} BufferPool;
+
+enum
+{
+    kPageSize                = PAGE_SIZE,
+    kPagesPerBufferPool      = 1,
```

Comment:
Any suggestions for the number of Pages to allocate per pool, given each page is 4KB? The current design (1 page at a time) allocates Buffers in groups of 31 per pool (accounting for overhead of `BufferPool`).